docs/operate/get-started/other-hardware/micro-module.md

```diff
@@ -204,158 +29,96 @@ To create a new module compatible with the Micro-RDK, follow these steps.
+            }))
+        }
+    }
 
-1. Navigate to your machine's page on the [Viam app](https://app.viam.com) and select the **CONFIGURE** tab.
-1. Select the part status dropdown to the right of your machine's name on the top of the page: {{<imgproc src="configure/machine-part-info.png" resize="500x" declaredimensions=true alt="Restart button on the machine part info dropdown" class="shadow" >}}
-1. Click the copy icon underneath **Machine cloud credentials**.
-   The Micro-RDK needs this JSON object, which contains your machine part secret key and cloud app address, to connect to the [Viam app](https://app.viam.com).
-1. Navigate to the directory of the project you just created.
-1. Create a new <file>viam.json</file> file and paste the machine cloud credentials from the Viam app in.
-1. Save the file.
+   ```
 
-### Error: failed to run custom build command for `esp32-explorer (/host)`
+   {{< /expand >}}
 
-This may occur for various reasons such as your machine cloud credentials, WiFi SSID, or password not being populated.
-Check that your machine cloud credentials are provided in your project directory as <file>viam.json</file> and ensure that your WiFi credentials are provided.
+   For more examples, see the [example module implementation walkthrough](https://github.com/viamrobotics/micro-rdk/blob/main/examples/modular-drivers/README.md).
 
-### Error: invalid value `460800` for `--before <BEFORE>`
+For further details on Micro-RDK development, including credentials management and developer productivity suggestions, please see the [development technical notes page on GitHub](https://github.com/viamrobotics/micro-rdk/blob/main/DEVELOPMENT.md).
 
-Change `"-b"` to `"-B` in the <file>Makefile</file>, as `"-B"` is the Baudrate config.
-Run the following commands to flash <file>esp32-server.bin</file> to your ESP32 microcontroller at a high baud rate, wait for 2 seconds, and observe the device's output:
+## Using Your Module
 
-```sh {class="command-line" data-prompt="$"}
-espflash write-bin 0x0 target/esp32-server.bin -B 460800  && sleep 2 && espflash monitor
-```
+To use your module in a project:
 
-You can find additional assistance in the [Troubleshooting section](/manage/troubleshoot/troubleshoot/).
+1. Follow the [Build your firmware](/operate/get-started/setup-micro/#build-your-firmware) workflow in a different directory from your module.
+   Be sure to add your module to the `dependencies` section of the project's `Cargo.toml` file.
+2. [Flash your firmware](/operate/get-started/setup-micro/#flash-your-esp32) to your ESP32 microcontroller.
+3. [Configure](/operate/get-started/setup-micro/#configure-and-test-your-machine) your machine to use the module.
 
-{{< snippet "social.md" >}}
+To update your firmware with a new version of your module without a physical connection to the ESP32 microcontroller, see [Over-the-air firmware updates](/operate/reference/viam-micro-server/manage-micro/#over-the-air-updates).
```

Comment:
Instead of this, I think we should just add a Step 5 above that says something like:

"Follow the [Build and flash custom firmware](link) flow to test your module on a running ESP32"

Reply:
I kinda want to keep a "Use your module" heading to make it super quickly scannable. But I have condensed all these three steps into one sentence under "Use your module."

Reply:
Hmm. Pushing back on this one more time: the reason I think it makes sense combined with the above flow is that when you're developing a module, iterating and testing is a crucial part. I think with it split out into a separate section this way, a user may not realize that's how they should test.

Reply:
Calling it "Test your module" sounds like a good middle ground